.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -142,12 +146,11 @@ jobs:
             cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
             python -m pip install ninja cmake~=3.24.0
             yum clean all -y
-            yum install centos-release-scl-rh -y
-            yum install devtoolset-11-gcc-c++ -y
-            source /opt/rh/devtoolset-11/enable -y
+            yum install devtoolset-12-gcc-c++ -y
+            source /opt/rh/devtoolset-12/enable -y
 
           CIBW_ENVIRONMENT: |
-            PATH="/opt/rh/devtoolset-11/root/usr/bin:$PATH" \
+            PATH="/opt/rh/devtoolset-12/root/usr/bin:$PATH" \
```

Comment:
We can likely use the GCC 13 toolkit here instead of 12, but should probably set LD_LIBRARY_PATH too.

Reply:
@mlxd, I'm exploring this option locally, where my code is more mature.
The code in this WIP PR was just an initial mapping.